src/logger.ts

```diff
@@ -0,0 +1,6 @@
+import { Logger } from "./logging/Logger";
+import { LOG_LEVEL } from "./logging/types";
+
+export const logger = new Logger(
+  window.isLoggingEnabled === true ? LOG_LEVEL.DEBUG : LOG_LEVEL.NONE
```

Comment:
mb instead of having `isLoggingEnabled` just set logLevel?

Reply:
Postponed to discuss the changes with the team